janusgraph-solr/src/main/java/org/janusgraph/diskstorage/solr/SolrIndex.java

```diff
@@ -668,12 +709,12 @@ public String buildQueryFilter(Condition<JanusGraphElement> condition, KeyInform
                     if(tokenizer != null){
```

Comment:
I can not post my comment higher but you forgot fieldname for CONTAINS_PREFIX and REGEX

Reply:
The unit tests fail when using fieldName for CONTAINS_PREFIX and CONTAINS_REGEX since I think we want to use the _t index not the _s index to search.

Reply:
Which one failed ?

Reply:
Looking back at this, I'm not sure where I should add CONTAINS_PREFIX and CONTAINS_REGEX. Since there are two indexes created for TEXTSTRING, we only use the _s index for HAS_CONTAINS, all other predicates would use the _t index.

Reply:
@davidclement90 can you let me know if this is still an issue?  I need to open a new PR but want to make sure I've got everything closed from this one before proceeding.  Thanks.